packages/block-editor/src/components/block-parent/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import {
+  getBlocksByClientId,
+  getBlockAttributes,
+  getBlockName,
+  getBlockType } from '@wordpress/blocks';
+import { image } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import BlockIcon from '../block-icon';
+import BlockTitle from '../block-title';
+
+/**
+ * Block parent component, displaying the hierarchy of the
+ * current block selection as a single icon to "go up" a level.
+ *
+ * @return {WPElement} Block Breadcrumb.
+ */
+const BlockParent = function() {
+  const { selectBlock, clearSelectedBlock } = useDispatch(
+		'core/block-editor'
+	);
+	const { clientId, parents, hasSelection } = useSelect( ( select ) => {
+		const {
+			getSelectionStart,
+			getSelectedBlockClientId,
+			getBlockParents,
+		} = select( 'core/block-editor' );
+		const selectedBlockClientId = getSelectedBlockClientId();
+		return {
+			parents: getBlockParents( selectedBlockClientId ),
+			clientId: selectedBlockClientId,
+			hasSelection: !! getSelectionStart().clientId,
+		};
+	}, [] );
+
+  const firstParentClientID = parents[parents.length - 1];
+
+  const { parentClientId, parentBlockType } = useSelect( ( select ) => {
+    const { getBlockName } = select( 'core/block-editor' );
+    const parentBlockName = getBlockName( firstParentClientID )
+    return {
+      parentBlockType: getBlockType( parentBlockName ),
+    };
+  }, [] );
+
+  if ( parents && parents.length ) {
+    return (
+      <div className="block-editor-block-parent" key={ firstParentClientID }>
+          <Button
+            className="block-editor-block-parent__button"
+            onClick={ () => selectBlock( firstParentClientID ) }
+            label={ __( 'Select parent block' ) }
```

Comment:
I'd try `Select {parentBlockName}`, as the icon alone is not sufficient information.

Reply:
Yup yup, I had to figure out how to put a variable in the translate function. Found some random examples and got it working.